src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,24 +1,116 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 /**
  * <p>Реалізувати свою HashMap, а саме методи `put(K key, V value)`, `getValue()` та `getSize()`.
  * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size;
+    private int buckets;
+    private Node[] table;
+
+    public MyHashMap() {
+        this.table = new Node[INITIAL_CAPACITY];
+        this.buckets = 0;
+        this.size = 0;
```

Comment:
what's the default value for int after initialization?

Reply:
0, fixed this

Reply:
> 16 is not an experimental number)How do you think, what for do we need this method?
We use this method to find an index of array (bucket) were we should put an element.
Index, from tests has not enough range to call resize() method. So to increase range of indexes i applied this operation.